xarray/core/formatting.py

```diff
@@ -253,8 +253,15 @@ def summarize_coord(name, var, col_width):
 
 
 def summarize_attr(key, value, col_width=None):
-    # ignore col_width for now to more clearly distinguish attributes
-    return u'    %s: %s' % (key, maybe_truncate(value))
+    """Summary for __repr__ - use ``X.attrs[key]`` for full value."""
+    key = u'    %s:' % unicode_type(key)
+    if col_width is not None:
+        key = pretty_print(key, col_width)
+    output = u'%s %s' % (key, unicode_type(value).replace(u'\t', u'    '))
```

Comment:
It's not uncommon to find numbers or timestamps in the attributes, but it would surprise me to find something with a multiline repr.  I'd still want to truncate it to one line in that case, but that's more of a personal preference I guess.